firestore-send-email/functions/src/index.ts

```diff
@@ -311,6 +311,7 @@ async function processWrite(change) {
           transaction.update(change.after.ref, {
             "delivery.state": "ERROR",
             error: "Message processing lease expired.",
```

Comment:
maybe just leave a note in the code that it is to be removed in the next major release.

Reply:
@russellwheatley Thanks Russell, agreed. Added error message, open to suggestions for any  / less detail on the comment.